client/app/stores/commentsStore.js

```diff
@@ -5,11 +5,10 @@ import reducers from '../reducers';
 import { initalStates } from '../reducers';
 
 export default props => {
-  const initialComments = props;
   const { $$commentsState } = initalStates;
   const initialState = {
     $$commentsStore: $$commentsState.merge({
-      $$comments: initialComments,
+      $$comments: props,
```

Comment:
This is bizarre.

1. Please back out this change. The old code is important for readability.
2. we should ignore the eslint rule here and with a comment to the github issue indicating the problem.
3. please put a link to this code in the eslint issue (actually react-plugin probably, not main eslint)

Thanks @josiasds